src/lib/__tests__/sortMenu-test.js

```diff
@@ -0,0 +1,176 @@
+import sortMenu from '../sortMenu';
+
+describe('sortMenu', () => {
+  it('modifies original array to follow the group, then name order provided', () => {
+    const initialArray = [
+      {props: {group: 'DEV', name: 'Inspector'}},
```

Comment:
I would consider making a helper here that takes in threes arrays of strings of form `GROUP/NAME` and does a split and does the comparison, just to have some very compact test cases below